src/Data/Json/JTable.purs

```diff
@@ -0,0 +1,72 @@
+module Data.Json.JTable
+  ( renderJTable, renderJTableDef
+  , JTableOpts(..), jTableOptsDefault
+  , ColumnOrdering(..), inOrdering, alphaOrdering
+  , TableStyle(..), noStyle, bootstrapStyle, debugStyle
+  ) where
+
+import Data.Json.JTable.Internal
+
+import Data.String (joinWith)
+import Data.Argonaut.Core
+import Data.Argonaut.JCursor
+import Text.Smolder.HTML (table, thead, tbody, tr, th, td, br, small)
+import Text.Smolder.HTML.Attributes (className)
+import Text.Smolder.Markup (Markup(..), MarkupM(..), Attributable, attribute, (!), text)
+import Data.Foldable (mconcat)
+
+-- type JPath = [String]
```

Comment:
actually, comment to show the type of JPath, since it is exported but not defined here